packages/clay-navigation/src/Vertical.tsx

```diff
@@ -154,6 +160,7 @@ export const ClayVerticalNav: React.FunctionComponent<IProps> = ({
 	items,
 	large,
 	spritemap,
+	...otherProps
```

Comment:
The `IProps` interface needs to extend from the `HTMLElement` interface, it looks like we do not have an `HTMLNavElement`.

Reply:
Solved after some internal conversation!